app/commands/user/authenticate_from_omniauth.rb

```diff
@@ -0,0 +1,67 @@
+class User
```

Comment:
Almost a complete copy of what we had in `V2`. I didn't just blindly copy pasted this, but instead added tests first then copied over the relevant bits of code until we get to almost the same code.

I excluded the following:
- Automatically joining the user to the track based on a session value. I'm unsure whether we want that, and we can always put that in a separate PR in the future.
- Validating handle uniqueness against the `UserTrack`. Handles are only unique across users. I looked at the DB fields and `user_tracks` has no handle column yet. Again, we can add this as a separate PR if needed.

Reply:
Great. Thanks @kntsoriano :)